ledger/participant-integration-api/src/main/resources/db/migration/oracle/V2__Status_to_CLOB.sql

```diff
@@ -0,0 +1,4 @@
+alter table participant_command_completions add ( temp clob );
+update participant_command_completions set temp=status_message;
+alter table participant_command_completions drop column status_message;
+alter table participant_command_completions rename column temp to status_message;
```

Comment:
@danielporterda you can just change the column type in the V1 script as we don't need to do data migration for these schemas on oracle (yet)

Reply:
fixed and pushed